static/test/javascripts-legacy/spec/common/football.spec.js

```diff
@@ -0,0 +1,118 @@
+define([
+    'helpers/injector'
+], function (Injector) {
+
+    describe('Football', function () {
+        var injector = new Injector();
+        var sut;
+        var sandbox;
+        var destroySpy;
+
+        function mockMatchListLive() {}
+        
+        beforeEach(function (done) {
+            document.body.innerHTML = '<img class="media-primary"></img><div class="js-football-meta"></div>';
+
+            sandbox = sinon.sandbox.create();
+
+            destroySpy = sandbox.spy();
+
+            mockMatchListLive.prototype.destroy = destroySpy;
+
+            injector.mock('lib/config', {
+                dateFromSlug: sinon.spy()
+            });
+
+            injector.mock('bean', {
+                on: sinon.spy()
+            });
+
+            injector.mock('common/modules/sport/football/tag-page-stats', {
+                tagPageStats: sinon.spy()
+            });
+
+            injector.mock('common/modules/sport/football/match-list-live', {
+                MatchListLive: mockMatchListLive
+            });
+
+            injector.mock('lib/page', {
+                isMatch: sandbox.spy(),
+                isCompetition: sandbox.spy(),
+                isLiveClockwatch: function (cb) {
+                    cb();
+                },
+                isFootballStatsPage: sandbox.spy()
+            });
```

Comment:
A point of style, but your `injector.mock(...)` calls can be combined into a hash:

```
injector.mock({
  'lib/config': {
    dateFromSlug: sinon.spy()
  },
  'bean': {
    on: sinon.spy()
  },
  // ...
})
```